packages/cli/package.json

```diff
@@ -29,19 +29,19 @@
     "test": "TZ=UTC jest"
   },
   "dependencies": {
+    "@celo/contractkit": "^0.1.6",
```

Comment:
How recent is this version? is this == as master? or was it a while ago?

There's a small change that makes master != to alfajores: GasPriceMinimun has `proposerFraction` and before was `infrastructureFraction`

Reply:
This is the version of contractkit I had to redeploy in order to get the CLI to work on Alfajores-- I think it was around October 3/4. Is it safe to have this contractkit version?

Reply:
i believe so